sys/net/gnrc/sock/gnrc_sock.c

```diff
@@ -90,8 +91,7 @@ ssize_t gnrc_sock_recv(gnrc_sock_reg_t *reg, gnrc_pktsnip_t **pkt_out,
     msg_t msg;
 
 #ifdef MODULE_FUZZING
```

Comment:
Should probably also be changed to `if defined(MODULE_FUZZING) && !defined(MODULE_SOCK_ASYNC_EVENT)`.

Reply:
Or even better

```C
if (IS_ACTIVE(MODULE_FUZZING) && !IS_ACTIVE(MODULE_SOCK_ASYNC_EVENT)) {
    …
}
```